Support additional command line options (#5)
bitnami/bitnami-docker-logstash: README.md

$ docker run -d --env LOGSTASH_CONF_FILENAME=my_config.conf -v /path/to/custom-conf-directory:/bitnami/logstash/config bitnami/logstash:latest
```

## Additional command line options

> Using the `LOGSTASH_EXTRA_ARGS` environment variable if you want to specify some more command line options such as `pipeline.workers` and `pipeline.batch.size`.

Reviewer: ```suggestion
In case you want to add extra flags to the Logstash command, use the LOGSTASH_EXTRA_ARGS variable. Example:
```

```console

> $ docker run -d --env LOGSTASH_EXTRA_ARGS=" -w 4 -b 4096" bitnami/logstash:latest

Reviewer: ```suggestion
docker run --name logstash \
  --env LOGSTASH_EXTRA_ARGS="-w 4 -b 4096" \
  bitnami/logstash:latest
```

Author: I think the format here should be consistent with the above.

Reviewer: You are completely right, @fengxsong. Let's remove this little space and we are done:

```suggestion
$ docker run -d --env LOGSTASH_EXTRA_ARGS="-w 4 -b 4096" bitnami/logstash:latest
```
